### DISCUSSION TITLE
MDC documentation page is not loading

### DISCUSSION THREAD

**Discussion by daamsie** [nuxt/content#1458] [General]
Getting a bunch of errors in the console when navigating to [the MDC page](https://content.nuxtjs.org/guide/writing/mdc)

And this error when refreshing the page (SSR) 

**Invalid value used as weak map key**

**Comment by farnabaz** [comment]
Thanks for the report @daamsie, This is fixed 🙂

**Reply by daamsie** [reply]
Awesome, thank you :)
